### PR TITLE
[Outlook] (quickstart) update unified manifest quickstart

### PR DESCRIPTION
**Pull request by Rick-Kirkham**
These changes make the following assumptions:

1. The changes won't be published until GA of unified manifest for Outlook, currently expected in late February.
2. At the time of GA, The Yo Office option for task pane will also provide a manifest choice.

The quick start has been edited so it will work for either manifest. 
The special quick start for the unified manifest, that used TypeScript, has been moved to the "/excludes" folder in case we ever want to use it again. It has been removed from the ToC.

### CODE REVIEW

File: docs/develop/json-manifest-overview.md
Comment (samantharamon) [inline docs/develop/json-manifest-overview.md:108]:
```suggestion
- [Build your first Outlook add-in](../quickstarts/outlook-quickstart.md).
```

**Review by samantharamon — APPROVED** [review]
Approving with a question. The other drafts include a link to the unified manifest quick start. Will these PRs be updated before they're merged or will the instructions be updated in a separate PR?

**Comment by Rick-Kirkham** [comment]
> Approving with a question. The other drafts include a link to the unified manifest quick start. Will these PRs be updated before they're merged or will the instructions be updated in a separate PR?

I'm redoing the PR for the quick starts. Because we're adding support for JavaScript with the unified manifest (that development is probably an email that I scheduled-sent for later this week), we'll have just one quick start and the user will be instructed to choose JavaScript and then their choice of XML or unified manifest. The existing unified manifest quick start is moving the /excludes folder. See this PR: https://github.com/OfficeDev/office-js-docs-pr/pull/4341

File: docs/quickstarts/outlook-quickstart.md
Comment (samantharamon) [inline docs/quickstarts/outlook-quickstart.md:48]: I can't leave a comment directly on the affected section, so adding this here.

Lines 61 - 66 ("Explore the project"): Include the file structure of an add-in project that uses the unified manifest (see [Explore the project](https://learn.microsoft.com/office/dev/add-ins/quickstarts/outlook-quickstart-json-manifest#explore-the-project)).

Line 68 ("Update the code"): We may need to add a note specifying that this section only applies to projects that use the XML manifest. I have a to-do to incorporate this JS code into the Outlook Yo Office template, but I might not be able to complete the work in time.

**Review by samantharamon — CHANGES_REQUESTED** [review]

File: docs/quickstarts/outlook-quickstart.md
Comment (Rick-Kirkham) [reply docs/quickstarts/outlook-quickstart.md:48]: I updated the **Explore the project** section.

The **Update the code** section applies regardless of the type of manifest that they chose. 

Darren and I have already updated the task pane project for JavaScript so that the user can choose either language with either type of manifest. I'm not clear on what your ToDo is, but I don't think it's needed.

Comment (samantharamon) [reply docs/quickstarts/outlook-quickstart.md:48]: Thanks, Ricky.

Ah, I forgot that the change you and Darren worked on included language selection. Thanks for clearing that up. My task involves adding the JS code in the "Update the code" section of the Outlook quick start to the Outlook Yo Office template, so that the user can run the add-in right after creating it. This would align the Outlook quick start with the other quick starts, such as Excel.

**Review by samantharamon — APPROVED** [review]

File: docs/quickstarts/outlook-quickstart.md
Comment (Rick-Kirkham) [reply docs/quickstarts/outlook-quickstart.md:48]: > Thanks, Ricky.
> 
> Ah, I forgot that the change you and Darren worked on included language selection. Thanks for clearing that up. My task involves adding the JS code in the "Update the code" section of the Outlook quick start to the Outlook Yo Office template, so that the user can run the add-in right after creating it. This would align the Outlook quick start with the other quick starts, such as Excel.

OK. I think I get it now. Step 3 of the "Update the code" section would be deleted because that code would be in the project when it is created.

**Pull request merged**
